testnet/Dockerfile

```
@@ -0,0 +1,38 @@
+# BUILD IMAGE ---------------------------------------------------------
+
+FROM rust:1.72.0-slim-bullseye AS builder
+
+# Using backports for go 1.19
+RUN echo 'deb http://deb.debian.org/debian bullseye-backports main' \
+    >> /etc/apt/sources.list
+
+# Dependecies for publishing documentation and building waku-bindings.
+RUN apt-get update && apt-get install -yq \
+    git clang etcd-client \
+    golang-src/bullseye-backports \
+    golang-doc/bullseye-backports \
+    golang/bullseye-backports
+
+WORKDIR /nomos
+COPY . . 
+
+RUN cargo build --release -p nomos-node --no-default-features --features libp2p
+RUN cargo build --release -p mixnode
+
+# NODE IMAGE ----------------------------------------------------------
+
+FROM bitnami/minideb:latest
+
+LABEL maintainer="augustinas@status.im"
+LABEL source="https://github.com/logos-co/nomos-node"
+LABEL description="Nomos node image"
```

You can avoid creating more layers than necessary by using just one `LABEL` directive:
```suggestion
LABEL maintainer="augustinas@status.im" \
      source="https://github.com/logos-co/nomos-node" \
      description="Nomos node image"
```